react/features/base/media/components/web/Video.js

```diff
@@ -149,13 +243,28 @@ class Video extends Component<Props> {
      * @returns {ReactElement}
      */
     render() {
+        const {
+            autoPlay,
+            className,
+            id,
+            // eslint-disable-next-line no-unused-vars
+            onVideoPlaying,
+            // eslint-disable-next-line no-unused-vars
+            videoTrack,
+            playsinline,
+            style,
+            ...eventHandlers
```

Comment:
This looks a bit dangerous if we add more props. Maybe be explicit and extract just the ones we care about to `eventHandlers` ? It will also save us those ugly eslint disabling comments ;-)